.github/workflows/build-and-deploy.yml

```diff
@@ -223,7 +223,7 @@ jobs:
           signing-password: ${{ secrets.SIGNING_PASSWORD }}
           maven-central-password: ${{ secrets.MAVEN_CENTRAL_PASSWORD }}
           should-run-codecov: false
-          should-deploy: ${{ contains('Linux', runner.os) && needs.ci-preparation.outputs.may-deploy }}
+          should-deploy: ${{ contains('Linux', runner.os) && needs.ci-preparation.outputs.may-deploy == 'true' }}
```

Comment:
why?

Reply:
@Zimbrando

Reply:
It is useless, I thought it was the reason my runs were failing. Resolved in https://github.com/AlchemistSimulator/Alchemist/pull/3120/commits/5a76ebfc7986557a01fc36247e28aff767f3fbb0